bench/sth_conc_bench_test.go

```diff
@@ -41,8 +41,7 @@ func sthWithPutConcurrency(b *testing.B, c int) func() (indexer.Interface, error
 		return storethehash.New(context.Background(), b.TempDir(),
 			indexer.BinaryValueCodec{},
 			c,
-			sth.GCInterval(30*time.Minute),
-			sth.GCTimeLimit(5*time.Minute),
+			sth.GCInterval(0),
```

Comment:
@gammazero is it realistic to have the GC disabled?

Reply:
I think so, because the first run happens 1 minute into startup time (as a special case) and may affect some benchmarks and not others depending on exactly when it starts and ends.  While it may not affect performance greatly or at all, it could throw off other metrics like memory allocs.

If we want to see if GC has any significant than a separate benchmark should be run, making sure GC is started and running during the benchmark.